1.0/tools/tests.md

```diff
@@ -0,0 +1,385 @@
+
+Below is an example of a simple XHR unit test suite for 
+[`<iron-ajax>`](https://elements.polymer-project.org/elements/iron-ajax).
+Check out Sinon's documentation for more in-depth examples. 
+
+{% highlight html %}
+<!-- create test fixture template -->
+<test-fixture id="simple-get">
+  <template>
+    <iron-ajax url="/responds_to_get_with_json"></iron-ajax>
+  </template>
+</test-fixture>
+<script>
+  suite('<iron-ajax>', function() {
+    var ajax;
+    var request;
+    var server;
+    var responseHeaders = {
+      json: { 'Content-Type': 'application/json' }
+    };
+    setup(function() {
+      server = sinon.fakeServer.create();
+      server.respondWith(
+        'GET',
+        /\/responds_to_get_with_json.*/, [
+          200,
+          responseHeaders.json,
+          '{"success":true}'
+        ]
+      );
+    });
+    teardown(function() {
+      server.restore();
+    });
+    suite('when making simple GET requests for JSON', function() {
+      setup(function() {
+        // get fresh instance of iron-ajax before every test
+        ajax = fixture('simple-get');
+      });
+      test('has sane defaults that love you', function() {
+        request = ajax.generateRequest();
+        server.respond();
+        expect(request.response).to.be.ok;
+        expect(request.response).to.be.an('object');
+        expect(request.response.success).to.be.equal(true);
+      });
+      test('has the correct xhr method', function() {
+        request = ajax.generateRequest();
+        expect(request.xhr.method).to.be.equal('GET');
+      });
+    });
+  });
+</script>
+{% endhighlight %}
+
+The example above uses Chai's [`expect`](http://chaijs.com/api/bdd/) assertion
+style. 
+{: .alert .alert-info }
+
+## Run a subset of tests {#test-subsets}
+
+To run a subset of tests, create an HTML file and call `loadSuites()`. When
+running `wct`, specify the path to the HTML file as the first argument 
+(for example, `wct test/my-test-subset.html`.
+
+{% highlight html %}
+<!doctype html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <script src=”../bower_components/webcomponentsjs/web-components-lite.js”></script>
+    <script src=”../bower_components/web-component-tester/browser.js”></script>
+  </head>
+  <body>
+    <script>
+      WCT.loadSuites([
+        'basic.html',
+        'async.html'
+      ]);
+    </script>
+  </body>
+</html>
+{% endhighlight %}
+
+The argument to `loadSuites()` should be an array of strings. Each string
+should be a relative URL to a test suite. You can configure your tests 
+using query strings in the URLs. See [Test Shadow DOM](#shadow-dom)
+for an example.
+
+## Asynchronous tests
+
+To create an asynchronous test, pass `done` as an argument to the test function
+and then call `done()` when the test is complete. The `done` argument is a
+signal to Mocha that the test is asynchronous. When Mocha runs the tests it 
+waits until it encounters the `done()` call, and eventually times out if it 
```

Comment:
"it does not encounter it" is a little funky. Also, we should identify what happens when the test times out.

Maybe: "it waits until the test code invokes the `done()` callback. If the `done()` callback isn't invoked, the test eventually times out and Mocha reports it as a failure."

We normally don't use parens after method names, but in this case I think they're worth including because the API is somewhat funky, and it might not be clear that the parameter you're passing (`done`) is a callback that's pre-defined by Mocha.